test/python/visualization/test_plot_histogram.py

```diff
@@ -13,15 +13,21 @@
 """Tests for plot_histogram."""
 
 import unittest
-import matplotlib as mpl
+from io import BytesIO
+from collections import Counter
+from PIL import Image
+from qiskit.tools.visualization import plot_histogram, HAS_MATPLOTLIB
+from .visualization import QiskitVisualizationTestCase
 
-from qiskit.test import QiskitTestCase
-from qiskit.tools.visualization import plot_histogram
 
+if HAS_MATPLOTLIB:
+    import matplotlib as mpl
```

Comment:
consider using lazy testers for this (see https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#checking-for-optionals), introduced in https://github.com/Qiskit/qiskit-terra/pull/7525/files